libshaderc_util/src/compiler.cc

```diff
@@ -103,20 +103,31 @@ spv_result_t DisassembleBinary(const std::vector<uint32_t>& binary,
   return result;
 }
 
+// Coverts a string to a vector of uint32_t by coping the content of a given
+// string to a vector<uint32_t> and return it. Appends '\0' at the end if extra
+// bytes are required to complete the last element.
+std::vector<uint32_t> ConvertStringToVector(std::string& str) {
```

Comment:
Should I pull it out from here and place it with some sort of 'util' functions? I think this function is only used here and it is not relevant any external code, except STL, so it might be fine to live here?